common/test/acceptance/tests/studio/test_studio_settings_details.py

```diff
@@ -161,3 +161,32 @@ def test_enable_entrance_exam_for_course(self):
             css_selector='span.section-title',
             text='Entrance Exam'
         ))
+
+    def test_entrance_exam_has_unit_button(self):
+        """
+        Test that entrance exam should be created after checking the 'enable entrance exam' checkbox.
```

Comment:
Going forward on other projects, please follow the documentation best practices for bok choy tests, listed on

https://openedx.atlassian.net/wiki/display/TNL/Code+Review+Guidelines+and+Gotchas (see bok choy section).